templates/inspec/examples/attributes/external_attributes.yml

```diff
@@ -9,4 +9,5 @@ gcp_location: europe-west2
 gcp_network_name: gcp-inspec-network
 gcp_organization_id: 190694428152
 gcp_storage_bucket_name: gcp-inspec-storage-bucket
-gcp_cloud_function_region: europe-west1
\ No newline at end of file
+gcp_cloud_function_region: europe-west1
+gcp_billing_account: billingAccounts/012345-567890-ABCDEF
```

Comment:
This is definitely a fake value. How does it get swapped out for testing?

Reply:
Via a config file in the inspec-gcp repo where you manually set this value (along with organization id, and some others)